BoyancyEngineTest/BoyancyEngineTest.ino

```diff
@@ -248,4 +248,5 @@ void loop() {
   loopCompass();
   //vt100DashLoop();
   loopStorage();
```

Comment:
hmm... we are calling both here, but we probably don't want to do that.
How about we comment out 'loopStorage();' for now?